### PR TITLE
Restructure Ecosystem

### PR DESCRIPTION
**Pull request by mileslucas**
I've added some missing packages and reorganized the ecosystem into its own page with grouped headers like "Data I/O", "Image Processing", etc.

I'm eager for any comments and would like input on any of the packages I've added (cc @sefffal I added DirectOrbits.jl and edited the AstroImages.jl bullets, feel free to edit either!).

### CODE REVIEW

**Comment by mileslucas** [comment]
Here is the preview

http://juliaastro.github.io/previews/PR20/

**Comment by sefffal** [comment]
This looks great!
For DirectOrbits, I recently registered that package under the new name PlanetOrbits.jl since it now supports RV orbital elements too.
Thanks!

File: src/ecosystem.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+- Wrapper for [CasaCore](http://casacore.github.io/casacore/)
+- Interface for CASA tables and measures
+
+### [emmt/OIFITS.jl](https://github.com/emmt/OIFITS.jl)
+
+**Support for OI-FITS**
+
+- Read and write OI-FITS optical interferometry data format
+
```
[the remark this answers is not in the document]
Comment (giordano) [reply src/ecosystem.md:39]: Sounds good, thanks!

File: src/ecosystem.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The joy of our community is the many astronomy and astrophysics packages availab
 
 **Wrapper for the Difmap program used in synthesis imaging**
 
-- Execute Difmap scripts, handle inputs/outputs and logs
+- Execute [Difmap](https://github.com/sabourke/difmap) scripts, handle inputs/outputs and logs
```
[the remark this answers is not in the document]
Comment (mileslucas) [reply src/ecosystem.md:50]: Will fix, thanks!

**Pull request merged**